build/automation/var/profile/task.mk

```diff
@@ -12,7 +12,5 @@ DOS_API_GATEWAY_URL := $(or $(DOS_API_GATEWAY_URL), "//")
 # ==============================================================================
 # Infrastructure variables (Terraform, Serverless, etc)
 
-TF_VAR_api_gateway_api_key_name := $(PROJECT_ID)-$(ENVIRONMENT)-api-key
-TF_VAR_nhs_uk_api_key_key := NHS_UK_API_KEY
```

Comment:
Can all these variables be re-added please

Reply:
Resolved